src/AutoUI/Annotations/SelectionAttribute.cs

```diff
@@ -2,13 +2,21 @@
 
 namespace DotVVM.AutoUI.Annotations;
 
+/// <summary>
+/// Indicates that the user will select a value from a list of SelectionType provided by ISelectionProvider&lt;SelectionType&gt;
```

Comment:
```suggestion
/// Indicates that the user will select a value from a list of SelectionType provided by <see cref="ISelectionProvider{SelectionType}" />
```

You can use this to avoid these XML entities